remote-messaging/remote-messaging-api/src/main/java/com/duckduckgo/remote/messaging/api/RemoteMessage.kt

```diff
@@ -56,18 +58,28 @@ sealed class Content(val messageType: MessageType) {
         val secondaryAction: Action,
     ) : Content(BIG_TWO_ACTION)
 
+    data class PromoSingleAction(
+        val titleText: String,
+        val descriptionText: String,
+        val placeholder: Placeholder,
+        val actionText: String,
+        val action: Action,
+    ) : Content(PROMO_SINGLE_ACTION)
+
     enum class MessageType {
         SMALL,
         MEDIUM,
         BIG_SINGLE_ACTION,
         BIG_TWO_ACTION,
+        PROMO_SINGLE_ACTION,
     }
 
     enum class Placeholder(val jsonValue: String) {
         ANNOUNCE("Announce"),
         DDG_ANNOUNCE("DDGAnnounce"),
         CRITICAL_UPDATE("CriticalUpdate"),
         APP_UPDATE("AppUpdate"),
+        MAC_AND_WINDOWS("NewForMacAndWindows"),
```

Comment:
Why so specific, ie. `NewForMacAndWindows`. Will this be **only** use for mac and windows announcements?
Can we call it something more generic?

Reply:
The placeholder name was already defined for the iOS implementation, and I believe it makes sense as it includes the Windows and Apple logos in it.